modules/bmo-editing-the-hostfirmwarecomponents-resource-of-a-provisioned-host.adoc

```diff
@@ -76,49 +78,49 @@ status:
 <1> To set a BIOS version, set the `name` attribute to `bios`.
 <2> To set a BIOS version, set the `url` attribute to the URL for the firmware version of the BIOS.
 <3> To set a BMC version, set the `name` attribute to `bmc`.
-<4> To set a BMC version, set the `url` attribute to the URL for the firmware verison of the BMC.
+<4> To set a BMC version, set the `url` attribute to the URL for the firmware version of the BMC.
 
 . Save the changes and exit the editor.
 
-. Get the host’s machine name:
+. Get the host's machine name by running the following command:
```

Comment:
See the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives) guidance on avoiding possessives for inanimate objects.

Reply:
Done.